WordPress/Classes/Models/Blog.m

```diff
@@ -147,6 +147,10 @@ - (NSString *)icon
 // Used as a key to store passwords, if you change the algorithm, logins will break
 - (NSString *)displayURL
 {
+    if (self.url == nil) {
+        return nil;
```

Comment:
I'm wondering if adding a `DDLogInfo` message here to log what went wrong might be helpful?

Reply:
Good idea! Just added.